### ISSUE TITLE
"Wildly inaccurate" render estimations in checktiles mode

### ISSUE THREAD

**Issue by jack-webb** [overviewer/Minecraft-Overviewer#1944]
When I start a render after terminating, the tile count and ETA will both be so off as to be totally useless. Kind of begs the question, _why show an estimate at all?_ There's a few issues on here that cite this as a problem, and after a short IRC conversation with CounterPillow, there seems to be people asking there too (me included...). There _is_ a warning message about these estimations being inaccurate, but mine were coming out up to 30 million years, so I didn't really make the connection. I'm guessing other people either missed the message or made the same mistake I did.

- **In checktiles mode, should overviewer show the tile count and/or ETA at all?** The warning seems kinda ineffective.
- **If not, what do should it be replaced with? Is there some broader heuristic we can calculate instead?** Otherwise, probably just an indeterminate spinner.

Actually removing the check in the observer when in checktiles mode seems reasonably straightforward, but I don't think that alone is a good solution.

<details><summary>Original IRC convo</summary>
```
[YOU]
Hi all, I saw some issues about inaccurate ETAs when resuming renders after terminating. Does this estimate fall under that or is it likely to be something else? `Rendered 55952 of 117281254663900.  0% complete.  ETA: 979691829h 09m 58s`

[CounterPillow]
the estimate will be wildly inaccurate
because it enters checktiles mode
I don't know why we even show an estimate
we do say that it's wildly inaccurate in the console, but we still get questions about it, so maybe we should just remove it in check-tiles mode

[YOU]
Makes sense, thanks. There's wildly, and then there's.. that. At least imo. Maybe at that point yeah, remove the ETA. doesn't really serve any useful purpose 
```
</details>

**Comment by ProkopRandacek** [comment]
I agree that the current state of the ETA for render after terminating is useless.

Can it be replaced with a ETA as if a full render was being done with a warning that this is a upper limit?
